stl/inc/algorithm

```diff
@@ -5536,7 +5536,8 @@ namespace ranges {
 
             // _Temp_count + 1 since we work on closed ranges
             const auto _Total_count = static_cast<iter_difference_t<_It>>(_Temp_count + 1);
-            auto _Result            = _Stable_partition_common_buffered(
+

```

Comment:
So clang-format adds some new lines, right?

Reply:
I added these newlines manually in order to prevent clang-format from aligning the `=` operators and then getting confused and emitting an over-120-column line (which is really a clang-format bug, it should never emit overly long lines). This seemed like the least ugly solution. (In one case I needed to manually wrap with `//` which is slightly uglier, but still better than disabling clang-format entirely.)